assembly/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+migrator:
+  rdbms-exporter:
+    auto-ddl: true
+  source:
+    jdbc-url: jdbc:h2:~/Downloads/c8_rdbms-source.db;TRACE_LEVEL_FILE=0;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
I was wondering removing this hard coded config. It would make no sense to add in memory h2 here. If someone runs the `examples/generate-runtime` this is where the files are created.